useful_audio:
  ros__parameters:
    debug: False
    USE_SILERO_VAD: True
    threshold: REPLACE
    DISABLE_KWS: False
    MAX_AUDIO_DURATION: 10
    WAKEWORD_TOPIC: REPLACE
